cmd/harbor/root/scanner/view.go

```diff
@@ -0,0 +1,56 @@
+// Copyright Project Harbor Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package scanner
+
+import (
+	"github.com/goharbor/harbor-cli/pkg/api"
+	"github.com/goharbor/harbor-cli/pkg/prompt"
+	"github.com/goharbor/harbor-cli/pkg/utils"
+	"github.com/goharbor/harbor-cli/pkg/views/scanner/view"
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+func ViewCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "view",
+		Short: "get scanner by id",
+		Args:  cobra.MaximumNArgs(1),
```

Comment:
same here usage could be better. 
```suggestion
		Use:   "view [scanner-id]",
		Short: "get scanner by id",
		Args:  cobra.MaximumNArgs(1),
```